ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -219,6 +219,20 @@ router bgp {{ router_bgp.as }}
 {%             endif %}
    {{ maximum_routes_cli }}
 {%         endif %}
+{%         if peer_group.missing_policy.address_family_all.direction_in_action.action is arista.avd.defined %}
+{%             if peer_group.missing_policy.address_family_all.direction_in_action.include is arista.avd.defined %}
+   neighbor {{ peer_group.name }} missing-policy address-family all include {{ peer_group.missing_policy.address_family_all.direction_in_action.include | arista.avd.natural_sort | join(" ") }} direction in action {{ peer_group.missing_policy.address_family_all.direction_in_action.action }}
+{%             else %}
+   neighbor {{ peer_group.name }} missing-policy address-family all direction in action {{ peer_group.missing_policy.address_family_all.direction_in_action.action }}
```

Comment:
we can use variables for big dict values to make it easy to read.

Reply:
it would be simpler if you just build a cli variable.